src/NHibernate/Tool/hbm2ddl/SchemaUpdate.cs

```diff
@@ -141,7 +141,8 @@ public void Execute(Action<string> scriptAction, bool doUpdate)
 			autoKeyWordsImport = autoKeyWordsImport.ToLowerInvariant();
 			if (autoKeyWordsImport == Hbm2DDLKeyWords.AutoQuote)
 			{
-				SchemaMetadataUpdater.QuoteTableAndColumns(configuration);
+				SchemaMetadataUpdater.Update(configuration, dialect);
```

Comment:
Need to be moved in some initialization method (in constructors indeed). Here it will needlessly reload the keywords at each `Execute` call after the first.